domains/windowsed.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.b4wkSB9hzfyZzAOO50m5Lmffz5ZpdPkeT6RBILTZckY94cbRa7BxRCVOAevnxu4zqonBcx017nf0Kgu1qP58hQV8rbPZM7bEeJwIuPh-gY_l1y_IkzE5Slq1CWvmFCBZvheUKYHFvPRcGGIqtxyKKaT6jcrJXv7xo_F2fldqbJ2NiJ88_u4WTXcfQ9CrPrPevocKfdesIrDwXnV2n2I0SpDV_QqycbmCcleRy3QzOpZMTHQyzAV_WA0B8OEkMftVSbiTr-S4WPRMpoZTRX7Er-l-ccC5HIs1c7P-twpXd4bphTsUBmWJL55dD0xq7vDQ2ouWq-o4ooqqZ3wmme-KvA.R7TKz12p_AeXBaX-7nEu9g.tjS2Ho_9HJkfxTIoz2MFeX34ptUfBAIh6vPSR_lMepN4Nz03-n_JxdpZW-5iEmDqBUUU82h7WteWeRKhyBkvi1syxFOnMRTBT4EveG-Ij9-skVlWjzeAQw-6WWRaGtDPfBSyZsTKyIb9iI2tVP7Kvw.jJ5K9-F9Rh-7DMlxCJ8yQA"
+        },
+    
+        "record": {
+            "A": ["76.76.21.21"]
```

Comment:
You need to just add Vercel TXT record, follow https://www.is-a.dev/docs/vercel/ for more information

Reply:
![image](https://github.com/user-attachments/assets/789776a1-2203-41c6-b203-ef363f61624c)

Reply:
huh lol?